src/main/kotlin/network/warzone/mars/player/commands/StatCommands.kt

```diff
@@ -90,6 +90,8 @@ class StatCommands {
                 .appendMultiLine { createLabelledStat("Wins", stats.wins, StatType.POSITIVE) }
                 .appendMultiLine { createLabelledStat("Losses", stats.losses, StatType.NEGATIVE) }
                 .appendMultiLine { createLabelledStat("Win %", stats.winPercentage, StatType.NEUTRAL) }
+                //.appendMultiLine { empty() }
+                //.appendMultiLine { createLabelledStat("Achievements", stats.achievements.size, StatType.NEUTRAL) }
```

Comment:
any reason we removed this?

Reply:
I don't remember why this was removed. I can add it back if that is what we want.